x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -134,13 +132,23 @@ export const MisconfigurationFindingsDetailsTable = memo(
         name: '',
         width: '5%',
         render: (rule: CspBenchmarkRuleMetadata, finding: MisconfigurationFindingDetailFields) => (
-          <EuiLink
+          <SecuritySolutionLinkAnchor
+            deepLinkId={SecurityPageName.cloudSecurityPostureFindings}
+            path={`configurations${getFindingsPageUrlFilteredByRuleAndResourceId(
```

Comment:
I'm not sure how this deeplinking works, but is it possible to avoid hardcoding `configurations` and `vulnerabilities` in the path and leverage these deepLinkIds? We do have these paths defined in our navigation, I guess it should be possible to use specific deep link ids instead of hardcoded path parts